### DISCUSSION TITLE
Doesn't show / doesn't allow - pop-ups. No ability to enter a password / etc

### DISCUSSION THREAD

**Discussion by Vsevolod-IT** [i3/i3#5046] [Q&A]
Hi all! Please forgive me, I have been looking for a long time for a solution to this problem, but could not. There are programs that must pop-up (to enter a password / code) for example Ubuntu Software requests the system password through this window, but it does not pop-up and thus blocks the continuation of the program (Ubuntu Software here only as an example). Very please help!!!
My system is ubuntu 20.04 (gnome) [if it matters]

**Comment by vincentbernat** [comment]
You should ensure you are running a policykit authentication agent, like `/usr/lib/policykit-1-gnome/polkit-gnome-authentication-agent-1`. Try to start one from the command line and check if you get the authentication popups to see if it fixes the problem.

**Reply by Vsevolod-IT** [reply]
Thanks a lot!  I apologize for the long answer, but this helped for Ubuntu Software but did not solve the problem with another program (forti client vpn) there, when connecting, a pop-up window appears for the code (mfa) it does not show
